operations/update.go

```diff
@@ -53,96 +59,109 @@ func Update() cli.Command {
 			if err != nil {
 				return errors.Wrap(err, "problem loading configuration")
 			}
-
-			client := conf.getRestCommunicator(ctx)
-			defer client.Close()
-
-			update, err := checkUpdate(client, false, forceUpdate)
-			if err != nil {
-				return err
-			}
-			if !update.needsUpdate || len(update.binaries) == 0 {
-				return nil
+			if conf.AutoUpgradeCLI && autoUpdate {
+				fmt.Printf("automatic CLI updates are already set up, %s flag is not necessary", autoUpdateFlagName)
```

Comment:
Since this prints to the user in the CLI, can this please be a complete sentence? (same with line 70)